src/main/webapp/nutrition.js

```diff
@@ -0,0 +1,71 @@
+//Use user input to search for possible foods
```

Comment:
```suggestion
/**
 * Use user input to search for possible foods.
 */
```
This is one of the standard ways to comment functions in JavaScript and this applies to the other functions you wrote as well. I omitted writing the arguments and return types. This is a good reference for how people write JavaScript at google https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments